### ISSUE TITLE
Readme needs updates

### ISSUE THREAD

**Issue by Zodiac1978** [lloc/Multisite-Language-Switcher#254]
I think this plugin is not necessary anymore:
http://wordpress.org/extend/plugins/wordpress-mu-domain-mapping/
(If you want to still link it, it can be shortened to https://wordpress.org/plugins/wordpress-mu-domain-mapping/)

Codex articles have moved:
https://codex.wordpress.org/Create_A_Network to
https://wordpress.org/documentation/article/create-a-network/

http://codex.wordpress.org/WordPress_in_Your_Language redirects to https://make.wordpress.org/polyglots/teams/
Not sure if this is the article you want ...

https://github.com/danieledesantis/jquery-language-detection is 9 years without updates
https://github.com/oncleben31/Multisite-Language-Switcher-Theme is archived/read-only since 2020.

Redirecting is a bad idea, as Google is following these redirects which could impact your indexing very badly:
https://wpml.org/documentation/getting-started-guide/language-setup/automatic-redirect-based-on-browser-language/how-browser-language-redirect-affects-google-indexing/

Unfortunately [Network Shared Media](http://wordpress.org/plugins/network-shared-media/) is closed.

Maybe these two could be helpful:
https://github.com/bueltge/multisite-global-media/
https://github.com/humanmade/network-media-library

**Comment by lloc** [comment]
Would you like to make the changes in the readme-file?

**Comment by Zodiac1978** [comment]
Yes, I'm happily providing the PR. We just need some decisions with which plugin recommendation you want to go in some cases.

Or we need to research the correct website ... I'll start with those things that are clear.

**Comment by Zodiac1978** [comment]
> http://codex.wordpress.org/WordPress_in_Your_Language redirects to https://make.wordpress.org/polyglots/teams/
Not sure if this is the article you want ...

Maybe this could be a better choice? Or is the Team overview page the place you want to link?
https://developer.wordpress.org/advanced-administration/before-install/in-your-language/

**Comment by lloc** [comment]
I must admit that I never really reviewed the readme.txt (just the necessary version dump whenever a new version came up). Your choice seems much butter indeed...

**Comment by patriciabt** [comment]
working on it with @lloc, thanks a lot @Zodiac1978

**Comment by patriciabt** [comment]
@Zodiac1978 and @lloc 

> Maybe these two could be helpful:
https://github.com/bueltge/multisite-global-media/
https://github.com/humanmade/network-media-library

those are also outdated now, right?

**Comment by Zodiac1978** [comment]
Yes, both are looking abandoned. :(

**Comment by lloc** [comment]
I'm fine with any suggestion/PR from your side @Zodiac1978 and @patriciabt.

Some considerations:

## WordPress MU domain mapping

I know some sites that still use the domain mapping plugin. I don't use it because I map secondary domains in Cloudflare. We can probably remove the part regarding any redirection.

## Redirection

I do that on some sites without penalty, but it may be out-of-scope for the Mutliste Language Switcher.

## Shared media

It's a pity but there might not be so many requests. So this is might be also out-of-scope for the plugin and could be therefore removed.

**Comment by lloc** [comment]
@Zodiac1978 @patriciabt Do we have additional things to work on for this ticket? Do we want to close it?

**Comment by Zodiac1978** [comment]
No need to close it IMO. Just needs a copy and decision for modifying or removing the outdated parts.

**Comment by lloc** [comment]
@Zodiac1978 @patriciabt I'd like to leave the decision to you. 😄
